app/gui/src/dashboard/hooks/projectHooks.ts

```diff
@@ -276,12 +280,21 @@ export function useCloseProjectMutation() {
 
       void client.cancelQueries({ queryKey })
     },
-    onSuccess: async (_, { type, id, parentId }) => {
+    onSuccess: async (_, { type, id, parentId, cloudProjectId }) => {
       await client.resetQueries({ queryKey: createGetProjectDetailsQuery.getQueryKey(id) })
       setProjectAsset(type, id, parentId, (asset) => ({
         ...asset,
         projectState: { ...asset.projectState, type: backendModule.ProjectState.closed },
       }))
+
+      // If the project runs in hybrid execution mode
+      if (cloudProjectId) {
```

Comment:
I case if user closes local project in offline mode, the mutation will fail because remoteBackend is not available. This _might_ lead to unexpected behavior.

Reply:
Not sure how to handle the failure properly. I'll leave a TODO comment

Reply:
i think typical for dashboard isa  FIXME with a link to a new issue in `cloud-v2`